### ISSUE TITLE
Upload materials from previous Sardana Workshops

### ISSUE THREAD

**Issue by reszelaz** [sardana-org/sardana-followup#17]
In the past we have organized several Sardana Workshops. Some time ago there was a plan to upload them to the wiki pages repository. But IMHO this repository is also a good place. Then, in the wiki, we could just provide links to the corresponding directories in this repo. What do you think?

These are the meetings:
* Barcelona 2013
* Grenoble 2014
* Hamburg 2015 (directory could be named: 20151006-Hamburg)
* Copenhagen 2016 (directory could be named: 20161020-Copenhagen)
* Barcelona 2017 (Sardana workshop for beginners at ICALEPCS2017 - these materials are already avialable in the sardana-org/sardana-training repository)

**Comment by teresanunez** [comment]
@reszelaz, I agree with you. I think that it is a good idea to upload them to this repo. Thanks.

**Comment by reszelaz** [comment]
Great @teresanunez. Do you think that you could still recover the presentations/materials from the one done at DESY? Thanks!

**Comment by teresanunez** [comment]
Hi @reszelaz, the talks from the DESY WS are now uploaded.
